network/p2p/p2p_signatures.go

```diff
@@ -17,8 +18,13 @@ func (n *p2pNetwork) BroadcastSignature(msg *proto.SignedMessage) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to marshal message")
 	}
+	topic, err := n.GetTopic(msg)
+	if err != nil {
+		return errors.Wrap(err, "failed to get topic")
+	}
 
-	return n.cfg.Topic.Publish(n.ctx, msgBytes)
+	n.logger.Debug("Broadcasting to topic", zap.Any("topic", topic), zap.Any("peers", topic.ListPeers()))
```

Comment:
not sure we need this log

Reply:
agree, will remove it

Reply:
@alonmuroch on sec thought, it is important for debugging